app/controllers/concerns/authorization_count_concern.rb

```diff
@@ -28,3 +28,8 @@ def delete_auth_count(request_id)
     session[:sp_auth_count].delete request_id
   end
 end
+
+def sign_in_duration
```

Comment:
this is defined outside of the concern aka it's making a method name at global scope, I don't think that']s what we want

Reply:
Definitely not! Good catch.